### ISSUE TITLE
Could you give me some advice on how to solve the issue I face

### ISSUE THREAD

**Issue by horngjason** [anuranbaka/OpenDTAM#23]
Hi anuranbaka, 

Thanks for your kindly contribution :) 
However, I am currently facing some problem while do "make" the file you provide as following:
![screenshot_make](https://cloud.githubusercontent.com/assets/804746/6742365/1d92065e-ce87-11e4-94a9-e81115bfdf34.png)

Here is the system setup:
- Ubuntu-14.04 64bit
- OpenCV-2.4.10
- Cuda toolkit-5.5
- libboost-1.49

And, here is the result of "cmake ..":
![screenshot_cmake](https://cloud.githubusercontent.com/assets/804746/6742358/0dd523d6-ce87-11e4-951d-4f9fb3e1caf6.png)

Also, the result of "cmake -L
![screenshot_cmake-l](https://cloud.githubusercontent.com/assets/804746/6742379/3751bdd2-ce87-11e4-8c7b-c2e98f2276d4.png)

Not sure... whether or not that I need to use OpenCV-3.0.0 instead of my current one (i.e., OpenCV-2.4.10)? Looking forward to your kindly help!

Best Regards,
Yao-Ling

**Comment by anuranbaka** [comment]
Those look like opencv 3 linkages. Which commit on which branch are you
building?

**Comment by horngjason** [comment]
Thanks for your prompt reply! I am currently trying to build the "master branch in the OpenDTAM - Cpp".

And, same as you said... 
I found either of these to linkages are shown in OpenCV-3.0.0, but they seems like not in the OpenCV-2.4.10.

However, this is quite strange to me because I know there are people who successfully ran your source code under OpenCV-2.4.9. 
Not sure why I was not able to make it.

**Comment by anuranbaka** [comment]
I would try the backport branch. The repo organization is currently a bit
borked, because I had to make the code compatible with OpenCV 3 for
release, so I tried to make the master run on that and backport the code to
2.4.9, but that was a huge logistical nightmare, so I have been working in
an experimental branch off the backport branch. Then other projects came up
and I haven't really been able to keep up with it. I'm still actively
developing, but on a combination of avindra's fork and my own code, so it's
a bit confusing.
